composer.json

```diff
@@ -17,7 +17,7 @@
     "require": {
         "php": ">=7.4",
         "commongateway/corebundle": "^1.1.92",
-        "common-gateway/zgw-bundle": "1.0.48"
+        "common-gateway/zgw-bundle": "1.0.50"
```

Comment:
is this the latest (/correct) version?

Reply:
It is a workable version. I didn't have the time today to check if it works with 1.1 and further. 1.0.50 is the version that fixes mongoDB incompatibility